### PR TITLE
Update change-email-address.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note to explain that people who donot have exchange license cannot use the domain for emails.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - following up on this. Please do the copy editor review for this PR. Thanks.

File: Office365-Admin/email/change-email-address.md [hunk 1]
```diff
@@ -91,8 +91,11 @@ You must have a global admin account to perform these steps.
 4. Follow the steps to confirm that you own your domain and to change your email address.
     
 You'll be guided to get everything set up correctly with your domain in Office 365.
+
+> [!NOTE]
+> If you're not using an exchange license, you cannot use the domain to send/receive emails from the o365 tenant.
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/change-email-address.md]:
```suggestion
> If you are not using an Exchange license, you cannot use the domain to send or receive emails from the Office 365 tenant.
```

**Comment by mypil** [comment]
@pebaum - PR has been copy edited and is ready for final review. This is based on issue #809

Thank you.

cc: @kenwith

**Pull request merged**